CTX.js

```diff
@@ -207,7 +161,7 @@ function contextObjectXMLToCOinS (text) {
 };
 
 function doImportFromText(text, showPrompt) {
```

Comment:
There is still some code for that: https://github.com/zotero/translators/pull/865/files#diff-62157bee6a2b307e1ba8ea136455ffadL232 . However, I can complete the items in the doImport in the same way as in the outside call from the web translator. Okay?